server/src/services/userService.ts

```diff
@@ -0,0 +1,24 @@
+import UserModel, { User } from '../models/User'
+
+class UserService {
+  async findUserByEmail(email: string) {
+    return UserModel.findOne({ email })
+  }
+
```

Comment:
You can't just store the users password as it is in the database. Please lookup password hashing and salting.